Makefile

```diff
@@ -165,10 +165,18 @@ taskmetadata-validator:
 ecr-execution-role-image:
 	$(MAKE) -C misc/ecr $(MFLAGS)
 
+gocyclo:
+	# Run gocyclo over all .go files in the agent, excluding vendor/ and testutils/ directories, and all *_test.go and *_mocks.go files
+	gocyclo -over 15 $(shell go list -f '{{$$p := .}}{{range $$f := .GoFiles}}{{$$p.Dir}}/{{$$f}} {{end}}' ./agent/... | grep -v /vendor/ | grep -v /testutils/ | grep -v _test\.go$ | grep -v _mocks\.go$)
+
+#TODO, create and add go vet target
+static-check: gocyclo
```

Comment:
can you also add a line somewhere before this to specify that `gocyclo` and `static-check` are specified `PHONY` targets? You could just add these in line 25 as well.

Reply:
Fixed.